lib/dynflow/config.rb

```diff
@@ -115,6 +115,14 @@ def method_missing(name)
       { 'hostname' => Socket.gethostname, 'pid' => Process.pid }
     end
 
+    config_attr :backup_deleted_plans, Algebrick::Types::Boolean do
```

Comment:
Do we even need this? Considering the `backup_dir` can be either a string or nil, thus functioning as a toggle as well